src/java/picard/analysis/RnaSeqMetrics.java

```diff
@@ -67,42 +67,43 @@
+    /** Fraction of the PF-aligned bases that encode ribosomal RNA sequence (RIBOSOMAL_BASES/PF_ALIGNED_BASES) */
     public Double PCT_RIBOSOMAL_BASES;
 
-    /** CODING_BASES / PF_ALIGNED_BASES */
+    /** Fraction of the PF-aligned bases that map to protein encoding regions of mRNA transcripts (CODING_BASES/PF_ALIGNED_BASES) */
     public double PCT_CODING_BASES;
 
-    /** UTR_BASES / PF_ALIGNED_BASES */
+    /**  Fraction of the PF-aligned bases that map to untranslated regions of mRNA transcripts (UTR_BASES/PF_ALIGNED_BASES) */
     public double PCT_UTR_BASES;
 
-    /** INTRONIC_BASES / PF_ALIGNED_BASES */
+    /**  Fraction of the PF-aligned bases that map within introns of a mRNA transcript (INTRONIC_BASES/PF_ALIGNED_BASES) */
     public double PCT_INTRONIC_BASES;
 
-    /** INTERGENIC_BASES / PF_ALIGNED_BASES */
+    /**  Fraction of the PF-aligned bases that map to intergenic (between genes) regions of a genome.  These regions are typically not associated with mRNA (INTERGENIC_BASES/PF_ALIGNED_BASES) */
     public double PCT_INTERGENIC_BASES;
 
-    /** PCT_UTR_BASES + PCT_CODING_BASES */
+    /**  Fraction of the PF-aligned bases map to either protein encoding or within the untranslated regions (UTR) of mRNA transcripts (PCT_UTR_BASES + PCT_CODING_BASES) */
     public double PCT_MRNA_BASES;
 
-    /** The percentage of bases mapping to mRNA divided by the total number of PF bases. */
+    /** The fraction of PF-bases that map to mRNA encoding regions */
     public double PCT_USABLE_BASES;
 
-    /** CORRECT_STRAND_READS/(CORRECT_STRAND_READS + INCORRECT_STRAND_READS).  0 if library is not strand-specific. */
+    /** Percentage of reads corresponding to mRNA transcripts which map to the correct strand of a reference genome 
+    (CORRECT_STRAND_READS)/(CORRECT_STRAND_READS + INCORRECT_STRAND_READS).  0 if library is not strand-specific. */
     public double PCT_CORRECT_STRAND_READS;
 
     /** The median CV of coverage of the 1000 most highly expressed transcripts. Ideal value = 0. */
     public double MEDIAN_CV_COVERAGE;
 
     /**
-     * The median 5 prime bias of the 1000 most highly expressed transcripts, where 5 prime bias is calculated per
-     * transcript as: mean coverage of the 5' most 100 bases divided by the mean coverage of the whole transcript.
+     * The median 5 prime bias of the 1000 most highly expressed transcripts. The 5 prime bias is calculated per
+     * transcript as: mean coverage of the 5'-most 100 bases divided by the mean coverage of the whole transcript.
      */
     public double MEDIAN_5PRIME_BIAS;
 
     /**
      * The median 3 prime bias of the 1000 most highly expressed transcripts, where 3 prime bias is calculated per
-     * transcript as: mean coverage of the 3' most 100 bases divided by the mean coverage of the whole transcript.
+     * transcript as: mean coverage of the 3'-most 100 bases divided by the mean coverage of the whole transcript.
```

Comment:
either use 5' and 3' or use 5 prime and 3 prime...I know that this isn't your original writing, but since these are being re-written, might as well go for consistency.